server/locks_controller_test.go

```diff
@@ -140,129 +135,23 @@ func TestDeleteLock_InvalidLockID(t *testing.T) {
 	responseContains(t, w, http.StatusBadRequest, "Invalid lock id \"%A@\"")
 }
 
+// TODO: I'm going to adapt the next 5 test cases to use some mock of DeleteLockCommand
```

Comment:
Am I OK to generate mocks for `DeleteLockCommand` so I can refactor these tests?

Reply:
Yes, looks like that will be too hard to instantiate a concrete instance of.